core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -240,31 +278,37 @@ object ZkUtils extends Logging {
   /**
    *  make sure a persistent path exists in ZK. Create the path if not exist.
    */
-  def makeSurePersistentPathExists(client: ZkClient, path: String) {
+  def makeSurePersistentPathExists(client: ZkClient, path: String, acls: List[ACL] = DefaultAcls) {
```

Comment:
I can see the argument for both approaches. From an internal usage perspective, I think I would lean towards not having the argument just yet because it makes it easier to reason about (we don't have to check all the calls to see if other ACLs are being used). If other users are also using these methods as a way to interact with ZK, however, they may find it useful to be able to set their own ACLs. I don't have a strong opinion either way, I think.